src/services/tx/tx-sender/sdk.ts

```diff
@@ -26,6 +26,37 @@ export const getAndValidateSafeSDK = (): Safe => {
   return safeSDK
 }
 
+async function switchOrAddChain(walletProvider: ConnectedWallet['provider'], chainId: string): Promise<void> {
+  const UNKNOWN_CHAIN_ERROR_CODE = 4902
```

Comment:
I assume this is Metamask specific. Do we need to handle this with other wallets as well?

Reply:
I guess with MM we already cover a lot of cases and not sure how WC would behave if we try to call `wallet_addEthereumChain`.